addons/account/views/product_views.xml

```diff
@@ -1,4 +1,16 @@
 <odoo>
+    <record id="product_view_kanban_catalog" model="ir.ui.view">
+        <field name="name">product.view.kanban.catalog.account</field>
+        <field name="model">product.product</field>
+        <field name="inherit_id" ref="product.product_view_kanban_catalog"/>
+        <field name="mode">primary</field>
+        <field name="arch" type="xml">
+            <xpath expr="//kanban" position="attributes">
+                <attribute name="js_class">account_product_kanban_catalog</attribute>
+            </xpath>
+        </field>
+    </record>
+
```

Comment:
Is it really necessary?
Can't you rely on patches instead and update the base feature in place ?
It'd be better if we didn't need a dedicated view depending on whether we support sections or not.